src/crs/projected/lambertconic.jl

```diff
@@ -0,0 +1,137 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    LambertConic{latₒ,lat₁,lat₂,Datum,Shift}
+
+Lambert Conic Conformal CRS with latitude origin `latₒ` standard parallels `lat₁` and `lat₂`,  `Datum` and `Shift`.
+"""
+struct LambertConic{latₒ,lat₁,lat₂,Datum,Shift,M<:Met} <: Projected{Datum,Shift}
+  x::M
+  y::M
+end
+
+LambertConic{latₒ,lat₁,lat₂,Datum,Shift}(x::M, y::M) where {latₒ,lat₁,lat₂,Datum,Shift,M<:Met} =
+  LambertConic{latₒ,lat₁,lat₂,Datum,Shift,float(M)}(x, y)
+LambertConic{latₒ,lat₁,lat₂,Datum,Shift}(x::Met, y::Met) where {latₒ,lat₁,lat₂,Datum,Shift} =
+  LambertConic{latₒ,lat₁,lat₂,Datum,Shift}(promote(x, y)...)
+LambertConic{latₒ,lat₁,lat₂,Datum,Shift}(x::Len, y::Len) where {latₒ,lat₁,lat₂,Datum,Shift} =
+  LambertConic{latₒ,lat₁,lat₂,Datum,Shift}(uconvert(m, x), uconvert(m, y))
+LambertConic{latₒ,lat₁,lat₂,Datum,Shift}(x::Number, y::Number) where {latₒ,lat₁,lat₂,Datum,Shift} =
+  LambertConic{latₒ,lat₁,lat₂,Datum,Shift}(addunit(x, m), addunit(y, m))
+
+LambertConic{latₒ,lat₁,lat₂,Datum}(args...) where {latₒ,lat₁,lat₂,Datum} = LambertConic{latₒ,lat₁,lat₂,Datum,Shift()}(args...)
+
+LambertConic{latₒ,lat₁,lat₂}(args...) where {latₒ,lat₁,lat₂} = LambertConic{latₒ,lat₁,lat₂,WGS84Latest}(args...)
+
+Base.convert(
+  ::Type{LambertConic{latₒ,lat₁,lat₂,Datum,Shift,M}},
+  coords::LambertConic{latₒ,lat₁,lat₂,Datum,Shift}
+) where {latₒ,lat₁,lat₂,Datum,Shift,M} = LambertConic{latₒ,lat₁,lat₂,Datum,Shift,M}(coords.x, coords.y)
+
+constructor(::Type{<:LambertConic{latₒ,lat₁,lat₂,Datum,Shift}}) where {latₒ,lat₁,lat₂,Datum,Shift} =
+  LambertConic{latₒ,lat₁,lat₂,Datum,Shift}
+
+lentype(::Type{<:LambertConic{latₒ,lat₁,lat₂,Datum,Shift,M}}) where {latₒ,lat₁,lat₂,Datum,Shift,M} = M
+
+==(
+  coords₁::LambertConic{latₒ,lat₁,lat₂,Datum,Shift},
+  coords₂::LambertConic{latₒ,lat₁,lat₂,Datum,Shift}
+) where {latₒ,lat₁,lat₂,Datum,Shift} = coords₁.x == coords₂.x && coords₁.y == coords₂.y
+
+isconformal(::Type{<:LambertConic}) = true
+
+# ------------
+# CONVERSIONS
+# ------------
+
+# inbounds(::Type{<:LambertConic}, λ, ϕ) = !(ϕ ≈ -π/2)
+inbounds(::Type{<:LambertConic}, λ, ϕ) = (-π/2 < ϕ <= π/2)
```

Comment:
should this be `ϕ > -π/2`? The upper bound is satisfied already by the constructor of `LatLon`